easybuild/easyconfigs/h/h5py/h5py-2.8.0-CrayGNU-18.07-python2-serial.eb

```diff
@@ -0,0 +1,37 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = "PythonPackage"
+
+name = 'h5py'
+version = '2.8.0'
+req_py_majver = '2'
+req_py_minver = '7'
+pyver = '%s.%s' % (req_py_majver, req_py_minver)
+py_rev_ver = '13.1'
+versionsuffix = '-python%s-serial' % req_py_majver
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface
+to the Hierarchical Data Format library, version 5. HDF5 is a versatile, mature
+scientific software library designed for the fast, flexible storage of enormous
+amounts of data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'verbose' : False}
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('PyExtensions', '%s.%s' % (pyver, py_rev_ver)),
+    ('cray-hdf5/1.10.2.0', EXTERNAL_MODULE),
+]
+
+prebuildopts  = ' python setup.py configure --hdf5-version=1.10.2 --hdf5=$HDF5_DIR && '
+
+options = {'modulename': 'os'}
```

Comment:
can you please delete this line? 

This is a workaround for the parallel version (sanity checks fail on the login nodes) and we already ran into trouble with the serial version because was not correctly tested during installation